src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+
+  const PERIOD_WITHOUT_DISCOUNT = 2;
+  const PERIOD_WITH_SMALL_DISCOUNT = 7;
+
+  let price = days * RENT_COST;
```

Comment:
Prefer using 'const' for the 'price' variable since it's not being reassigned, but rather a new value is returned each time.